src/Compatibility/Microsoft.DotNet.PackageValidation/ApiCompatRunner.cs

```diff
@@ -25,8 +25,8 @@ public class ApiCompatRunner
         private readonly string _rightPackagePath;
         private readonly bool _isBaselineSuppression;
 
-        public ApiCompatRunner(bool enableStrictMode,
-            CompatibilityLoggerBase log,
+        public ApiCompatRunner(CompatibilityLoggerBase log,
+            bool enableStrictMode,
```

Comment:
NIT: Any particular reason for the param reordering?

Reply:
Yes, I'm locally experimenting with leveraging an IoC container to better test components like the ApiCompatRunner without relying on full integration testing and specifying the type dependencies first, aids to that scenario. I will talk more about this in the meeting that I sent out for next Monday.